src/datomic_toolbox/core.clj

```diff
@@ -1,20 +1,31 @@
 (ns datomic-toolbox.core
   (:require [clojure.java.io :as io]
             [clojure.walk :as walk]
-            [datomic.api :as d]
-            [turbovote.resource-config :refer [config]])
+            [datomic.api :as d])
   (:refer-clojure :exclude [partition]))
 
+(def default-uri (atom nil))
+(def default-connection (atom nil))
+(def default-partition (atom nil))
+
+(defn configure! [{:keys [uri partition]}]
+  (reset! default-uri uri)
+  (reset! default-partition partition))
+
+(defn uri [] @default-uri)
+
 (defn connection []
-  (d/connect (config [:datomic :uri])))
+  (let [conn (d/connect (uri))]
+    (reset! default-connection conn)
```

Comment:
Ah, that's true. That resulted from my (still under discussion?) change to use fns everywhere instead of accessing the atoms directly. So yes, one of those two things should change (either the atom goes away or we replace atom-reading fns with direct atom derefs everywhere).